### PR TITLE
Enable smooth scroll

### PR DESCRIPTION
**Pull request by lmeier**
## What?

This enables smooth scroll on the radar docs.

## Why?

Better UX!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,11 @@ npm run start
 
 This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
 
+If you have problem starting, try running with node 16. With nvm,
+```
+nvm use v16.18.1
```
Comment (jsani-radar) [inline README.md:20]: interesting, I'm currently using 14 and haven't had any issues - guess it makes sense to upgrade to latest regardless, and 18 just came out

**Review by jsani-radar — COMMENTED** [review]
do you mind attaching a gif / video of the improved scrolling behavior?

**Comment by jsani-radar** [comment]
hmm noticed some slightly weird behavior, video below

basically, if you navigate to a different page, the content changes but the scroll still occurs to the top of the subsequent page - is there any way to disable the scroll when navigating away? might be some quirk with how docusaurus renders the html

https://www.loom.com/share/db24e894ae6e4d42acd064c38f7127bd

File: README.md (hunk 1 above)
Comment (kochis) [reply README.md:20]: Yeah, would like to better understand what was actually failing before committing this to the docs (this is a public repo)

Comment (lmeier) [reply README.md:20]: On node 19 (default on my system), it fails to start with
```
Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:71:19)
    at Object.createHash (node:crypto:140:10)
    at BulkUpdateDecorator.hashFactory (/Users/liam.meier/dev/documentation/node_modules/webpack/lib/util/createHash.js:145:18)
    at BulkUpdateDecorator.digest (/Users/liam.meier/dev/documentation/node_modules/webpack/lib/util/createHash.js:80:21)
    at /Users/liam.meier/dev/documentation/node_modules/webpack/lib/ChunkGraph.js:1494:51
    at RuntimeSpecMap.provide (/Users/liam.meier/dev/documentation/node_modules/webpack/lib/util/runtime.js:495:33)
    at ChunkGraph._getModuleGraphHashBigInt (/Users/liam.meier/dev/documentation/node_modules/webpack/lib/ChunkGraph.js:1489:37)
    at /Users/liam.meier/dev/documentation/node_modules/webpack/lib/ChunkGraph.js:1517:27
    at RuntimeSpecMap.provide (/Users/liam.meier/dev/documentation/node_modules/webpack/lib/util/runtime.js:495:33)
    at ChunkGraph._getModuleGraphHashWithConnections (/Users/liam.meier/dev/documentation/node_modules/webpack/lib/ChunkGraph.js:1516:41)
    at ChunkGraph.getModuleGraphHash (/Users/liam.meier/dev/documentation/node_modules/webpack/lib/ChunkGraph.js:1460:11)
    at NormalModule.updateHash (/Users/liam.meier/dev/documentation/node_modules/webpack/lib/Module.js:713:26)
    at NormalModule.updateHash (/Users/liam.meier/dev/documentation/node_modules/webpack/lib/NormalModule.js:1311:9)
    at Compilation._createModuleHash (/Users/liam.meier/dev/documentation/node_modules/webpack/lib/Compilation.js:3237:10)
    at Compilation.createModuleHashes (/Users/liam.meier/dev/documentation/node_modules/webpack/lib/Compilation.js:3209:10)
    at /Users/liam.meier/dev/documentation/node_modules/webpack/lib/Compilation.js:2469:11
```

**Comment by lmeier** [comment]
> 

Hmmm, that's undesirable, let me see if I can get a fix.

File: README.md (hunk 1 above)
Comment (jsani-radar) [reply README.md:20]: perhaps since 19 is beta / not LTS - if it works with 14 and 18 we can probably omit this instruction

Comment (kochis) [reply README.md:20]: Odd-numbered node versions are considered "non-stable" so maybe thats it https://stackoverflow.com/a/14791204

Comment (jsani-radar) [reply README.md:20]: yup, if it works with any stable version, I'd rather omit this line

**Comment by lmeier** [comment]
Closing for now, per direction from design

**Pull request closed without merging**